File: very-easy/desafio-03.js
// Escreva uma função que recebe um array e retorne um novo array com todas as posições invertidas do original sem alterá-lo. Não utilize os métodos como reverse, map, forEach, etc

const reverseArray = array => {
    const arrayReverse = []
    for (let lenght = array.length - 1; lenght >= 0; lenght--)
        arrayReverse.push(array[lenght])

    return arrayReverse
}
console.log(reverseArray(["oh", "hi", "mark"]));
// console.log(reverseArray([60, 50, 40, 30, 20, 10]))

const reverseArray2 = array => {
    const arrayReverse2 = [];
    for (let i = 0; i < array.lenght; i++) {
        arrayReverse2[i] = array[array.lenght - 1 - i]
    }
    return arrayReverse2
}

console.log(reverseArray2([60, 50, 40, 30, 20, 10]))

File: very-easy/desafio_03.py
def reverse_array(array):
    """Escreva uma função que recebe um array e retorne um novo array com todas as posições invertidas do original sem alterá-lo. Não utilize os métodos como reverse, map, forEach, etc"""
    indice = 0
    length = len(array) - 1
    array_reverse = []
    while length >= indice:
        # array_reverse.append(length_array)
        array_reverse.append(array[length])
        length -= 1
    return array_reverse


print(reverse_array([10, 72, 35, 4, 5, 66]))
print(reverse_array([30, 72, 35, 4, 5, 66]))
print(reverse_array([60, 72, 50, 40, 30, 20, 10]))
